src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindCommand.java

```diff
@@ -61,5 +61,12 @@ public record Options(
               description = "Include similarity function score in response.",
               type = SchemaType.BOOLEAN,
               implementation = Boolean.class)
-          boolean includeSimilarity) {}
+          boolean includeSimilarity,
+
+      // return vector embedding used for ANN sorting
+      @Schema(
+              description = "Return vector embedding used for ANN sorting.",
+              type = SchemaType.BOOLEAN,
+              implementation = Boolean.class)
```

Comment:
Use of `Boolean.class` is probably fine for "implementation" (and used already above for "includeSimilarity") but technically type for primitive `boolean` is `Boolean.TYPE`.